server/jwt_test.go

```diff
@@ -4890,3 +4890,87 @@ func TestJWTResponseThreshold(t *testing.T) {
 		}
 	})
 }
+
+func TestJWTQueuePermissions(t *testing.T) {
+	aExpKp, aExpPub := createKey(t)
+	aExpClaim := jwt.NewAccountClaims(aExpPub)
+	aExpJwt := encodeClaim(t, aExpClaim, aExpPub)
+	newUser := func(t *testing.T, permType string) string {
+		ukp, _ := nkeys.CreateUser()
+		seed, _ := ukp.Seed()
+		upub, _ := ukp.PublicKey()
+		uclaim := newJWTTestUserClaims()
+		uclaim.Subject = upub
+		switch permType {
+		case "allow":
+			uclaim.Permissions.Sub.Allow.Add("foo.> *.dev")
+		case "deny":
+			uclaim.Permissions.Sub.Deny.Add("foo.> *.dev")
+		}
+		ujwt, err := uclaim.Encode(aExpKp)
+		require_NoError(t, err)
+		return genCredsFile(t, ujwt, seed)
+	}
+	confFileName := createConfFile(t, []byte(fmt.Sprintf(`
+		port: -1
+		operator = %s
+		resolver = MEMORY
+		resolver_preload = {
+			%s : %s
+		}`, ojwt, aExpPub, aExpJwt)))
+	defer os.Remove(confFileName)
+	opts, err := ProcessConfigFile(confFileName)
+	if err != nil {
+		t.Fatalf("Received unexpected error %s", err)
+	}
+	errChan := make(chan error, 1)
```

Comment:
Add this:
```
opts.NoLog, opts.NoSigs = true, true
```
to avoid the test printing the server's log/output.

Reply:
done